stdlib/system/src/main/java/org/ballerinalang/stdlib/system/utils/SystemConstants.java

```diff
@@ -55,7 +55,7 @@
     static final String PROCESS_FIELD = "ProcessField";
 
     // System error codes
-    public static final String PROCESS_EXEC_ERROR = "{ballerina/system}ProcessExecError";
+    public static final String PROCESS_EXEC_ERROR = "{ballerina/system}InvalidOperationError";
```

Comment:
Are we using this constant anywhere? Shall we rename it?

Reply:
This is a temporary fix for a patch release. I did it this way to minimize the changes. I introduced a new error type called `ProcessExecError` in the master branch.